googlemock/docs/cheat_sheet.md

```diff
@@ -326,8 +326,7 @@ The `argument` can be either a C string or a C++ string object:
 `ContainsRegex()` and `MatchesRegex()` take ownership of the `RE` object. They
 use the regular expression syntax defined
 [here](../../googletest/docs/advanced.md#regular-expression-syntax).
-`StrCaseEq()`, `StrCaseNe()`, `StrEq()`, and `StrNe()` work for wide strings as
-well.
+All but them work for wide strings.
```

Comment:
This sentence isn't grammatically correct, and I'm not sure what it's intended to mean.

Reply:
Thanks for the review.
Let me explain what I meant. [This source](https://www.collinsdictionary.com/dictionary/english/all-but) says "all but" means "all except". If you take a look at the [table](https://github.com/google/googletest/blob/1f2e2e8059540247537e18cdab33bc77259273af/googlemock/docs/cheat_sheet.md#string-matchers), you will find my text in close adjacency, so my references:
* All = all these matches from the table (i.e. ContainsRegex(string), EndsWith(suffix), HasSubstr(string), MatchesRegex(string), StartsWith(prefix), StrCaseEq(string), StrCaseNe(string), StrEq(string) and StrNe(string))
* them = specific matches from the preceding sentence

So by "All but them work for wide strings" I meant "Take every string, remove ContainsRegex and MatchesRegex and Bob's your uncle; you're left with the ones that work with wide strings".

I'm not a native, but hopefully now you have a clear understanding of what I wanted to express. If I'm still wrong, please tell me :) You're welcome to provide me with a suggestion too.

Reply:
I am trying to come up with the grammatical rule that explains why this is incorrect. Even as a native speaker I don't know why this is incorrect... but it does sound strange. The best I can do is say that "them" sounds awkward here. The word "these" would be okay, but I'd prefer being explicit.

Do `StartsWith` and `EndsWith` work with wide strings too? If so, then perhaps

"All of these matchers, except `ContainsRegex` and `MatchesRegex` work for wide strings as well."

If they don't, then I think it's fine as is.

Reply:
They do, I'll take your version.
Hmm, quite interesting. Thanks for taking the time ;)